packages/data-context/src/actions/WizardActions.ts

```diff
@@ -146,23 +146,30 @@ export class WizardActions {
   async scaffoldTestingType () {
     const { currentTestingType, wizard: { chosenLanguage } } = this.ctx.coreData
 
-    assert(currentTestingType)
-    assert(chosenLanguage)
+    if (!currentTestingType || !chosenLanguage) {
+      return
+    }
```

Comment:
Commented on the Jira ticket. I agree it is weakened but a lot of this stuff is wrong in general and need to be cleaned up in better places than is in scope for getting this working.

Reply:
I think we're overhauling this anyway with a lot of the other work that's going on in Lachlan's PR. I'll add a note/TODO here.

Reply:
Is this change necessary though? I'd rather see it undone here while we're talking about it than in a future PR that may or may not catch this. 'Cause otherwise we're dealing with a function that decides if itself is called until a future date where these cleanups are a priority.

Reply:
Yes, the changes here making things work properly exposed legit bugs that I spent way too much time debugging before deciding it wasn’t worth the effort to fix because I’m ripping this all up in my next PR anyway

Reply:
@flotwig if it helps, I added an explicit comment in the ticket where I'll address this: https://cypress-io.atlassian.net/browse/UNIFY-1256?focusedCommentId=17678